flang/test/Lower/OpenMP/threadprivate-real-logical-complex-derivedtype.f90

```diff
@@ -22,26 +22,26 @@ module test
 
 contains
   subroutine sub()
```

Comment:
Will adding a "CHECK-LABEL" here improves time?

Reply:
Thanks. That seems to work fine. I will update the patch.

Reply:
updated.